pkg/router/controller/host_admitter.go

```diff
@@ -4,12 +4,11 @@ import (
 	"fmt"
 
 	"github.com/golang/glog"
+	routeapi "github.com/openshift/origin/pkg/route/apis/route"
+	"github.com/openshift/origin/pkg/router"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/watch"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
-
-	routeapi "github.com/openshift/origin/pkg/route/apis/route"
-	"github.com/openshift/origin/pkg/router"
 )
```

Comment:
Why these lines are moved? This is not related to this change and also I don't think we want to do that. This is what we follow in other places:
```
import (
standard imports
+ blank
k8s imports
+ blank
openshift imports
+ blank
Other imports
)
Except standard imports, other imports may be in any order.
```